org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/commands/LanguageCommand.java

```diff
@@ -27,6 +27,8 @@
  * Represents a command that loads a Spoofax language.
  */
 public class LanguageCommand implements IReplCommand {
+    private static final String[] ARCHIVES =
+        { ".zip", ".jar", ".tar", ".tgz", ".tbz2", };
```

Comment:
See f09c082075b7c94632d18e7ff5723d019998a7e2 for an attempt. @gfokkema do you think this is enough? Should there be a test for simply having folders too, or any other supported file system?